### DISCUSSION TITLE
Loading mesh from array

### DISCUSSION THREAD

**Discussion by dthillaithevan** [nschloe/meshio#1270] [General]
Hi, 

I am trying to load a 3D tetra (four nodes per elem) mesh that I generated in Matlab using the element and node definitions.

```
import numpy as np
import meshio
nodes = np.loadtxt('nodes.csv', delimiter =',').T 
elem = np.loadtxt('elements.csv', delimiter =',').T
cells = [("tetra", elem)]
meshio.write_points_cells('output.vtu',nodes,cells) 
```

It seems that the final output mesh has the wrong element connections - [this](https://ibb.co/ccZ96Np) is what the mesh looks like in Matlab, and [here](https://ibb.co/HVGMQn0) is the .vtu mesh. 

Does anyone have any ideas as to what could be causing this? 

Thanks!

**Comment by nschloe** [comment]
Perhaps a base-0 vs base-1 index issue?

**Reply by dthillaithevan** [reply]
Yeah you are right, thank you!!
